editor/src/components/canvas/canvas-strategies/draw-to-insert-strategy.spec.browser2.tsx

```diff
@@ -191,6 +215,73 @@ describe('Inserting into absolute', () => {
     )
   })
 
+  it('Click to insert with default size', async () => {
+    const renderResult = await renderTestEditorWithCode(inputCode, 'await-first-dom-report')
+    await startInsertMode(renderResult.dispatch)
+
+    const targetElement = renderResult.renderedDOM.getByTestId('bbb')
+    const targetElementBounds = targetElement.getBoundingClientRect()
+    const canvasControlsLayer = renderResult.renderedDOM.getByTestId(CanvasControlsContainerID)
+
+    const point = slightlyOffsetWindowPointBecauseVeryWeirdIssue({
+      x: targetElementBounds.x + 65,
+      y: targetElementBounds.y + 55,
+    })
+
```

Comment:
I think if you put here a mouse move event dispatch and then check the highlightedViews, that would be a pretty cheap way to check that we are highlighting the target parent. maybe it's also worth making a specific test that shows an edge case in which highlight differs from regular select mode highlight